erica/erica_legacy/pyeric/eric_errors.py

```diff
@@ -420,6 +421,11 @@ class EricAlreadyRevokedError(EricTransferError):
     """
     ERROR_CODE = 11
 
+    # Overwrite initaliser to set custom res_code
+    def __init__(self, res_code=None, eric_response=None, server_response=None, server_err_msg=None):
```

Comment:
Parameter res_code not needed right? It will always be the 8 (in your case) or the 4 (in our case)